docs/syntax/kbd.md

```diff
@@ -0,0 +1,117 @@
+# Keyboard shortcuts
+
+You can represent keyboard keys and shortcuts in your documentation using the `{kbd}` role. This is useful for showing keyboard commands and shortcuts in a visually consistent way.
+
+## Basic usage
+
+To display a keyboard key, use the syntax `` {kbd}`key-name` ``. For example, writing `` {kbd}`Enter` `` will render as a styled keyboard key.
+
+::::{tab-set}
+
+:::{tab-item} Output
+Press {kbd}`Enter` to submit.
+:::
+
+:::{tab-item} Markdown
+```markdown
+Press {kbd}`Enter` to submit.
+```
+:::
+
+::::
+
+## Keyboard combinations
+
+You can represent keyboard combinations by joining multiple `{kbd}` roles with a plus sign (+).
+
+::::{tab-set}
+
+:::{tab-item} Output
+{kbd}`ctrl` + {kbd}`C` to copy text.
+
+{kbd}`Shift` + {kbd}`Alt` + {kbd}`F` to format the document.
```

Comment:
Could we support ``{kbd}`Shift + Alt + F` ``?

Reply:
++ yep most configuration files use named keys like that e.g:

https://github.com/koekeishiya/skhd/issues/1

Same of e.g vim where you can use `<SPACE>` as literals.

Reply:
We will need `CMD` and `WIN`

Reply:
https://github.com/elastic/docs-builder/pull/1537/commits/634768bfb6311ad6a9e6588c9be92a0fa448a275